app/screens/receive-bitcoin-screen/receive-wrapper.tsx

```diff
@@ -87,10 +84,7 @@ const ReceiveWrapperScreen = () => {
     skip: !isAuthed,
   })
 
-  useRealtimePriceQuery({
-    fetchPolicy: "network-only",
-    skip: !useIsAuthed(),
-  })
+  useRealtimePriceWrapper()
```

Comment:
Do you want this to refresh every time a switch between btc and usd happends?

Reply:
> Do you want this to refresh every time a switch between btc and usd happends?

is that a suggestion or a question is that is currently the case?